build.gradle.kts

```diff
@@ -0,0 +1,58 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+group = "org.jetbrains.research.ml.pluginUtilities"
```

Comment:
`ml4se`?)
But I think we don't have concrete artifacts. For example, I use only `org.jetbrains.research`.